test/acts_as_mappable_test.rb

```diff
@@ -293,7 +293,7 @@ def test_ip_geocoder_exception
   end
 
   def test_address_geocode
-    Geokit::Geocoders::MultiGeocoder.expects(:geocode).with('Irving, TX').returns(@location_a)
+    Geokit::Geocoders::MultiGeocoder.expects(:geocode).with('Irving, TX').twice.returns(@location_a)
```

Comment:
Line is too long. [100/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.